addons/crm/tests/test_crm_lead.py

```diff
@@ -370,6 +373,14 @@ def test_crm_lead_stages(self):
         self.assertEqual(lead.probability, 100.0)
         self.assertEqual(lead.stage_id, self.stage_gen_won)  # generic won stage has lower sequence than team won stage
 
+    @users('user_sales_leads')
+    @freeze_time("2012-01-14")
+    def test_crm_lead_lost_date_closed(self):
+        self.assertFalse(self.lead_1.date_closed, "Initially, closed date is not set")
```

Comment:
Note that you have to do a with_user(self.env.user) to browse self.lead_1 with current user. Otherwise it is still set to admin I think.

Reply:
Does the test has to be done with the user if it is just to check that the date_closed is set?